scripts/circle-release-next-packages.sh

```diff
@@ -31,11 +42,14 @@ else
   echo $'\nGit status:'
   git status -s
 
+
   echo $'\nBuilding packages'
   yarn packages:build
 
   prapare_version_commit
 
+  unpublish_previous_canary
```

Comment:
using `canary` here as this is what we are going to use when https://github.com/grafana/grafana/pull/18195 is merged